### PR TITLE
DD-528 Adapt metadata export to show new license info

### PR DESCRIPTION
**Pull request by mderuijter**
having a hard time locating metadata export bug, where some export types fail.

### CODE REVIEW

File: src/main/java/edu/harvard/iq/dataverse/api/imports/ImportDDIServiceBean.java [hunk 1]
```diff
@@ -1179,11 +1179,11 @@ private void processDataAccs(XMLStreamReader xmlr, DatasetVersionDTO dvDTO) thro
                     if (NOTE_TYPE_TERMS_OF_USE.equalsIgnoreCase(noteType) ) {
                         if ( LEVEL_DV.equalsIgnoreCase(xmlr.getAttributeValue(null, "level"))) {
                             String termOfUse = parseText(xmlr, "notes");
-                            if (termOfUse != null && termOfUse.trim().equals("CC0 Waiver") ) {
+                            /*if (termOfUse != null && termOfUse.trim().equals("CC0 Waiver") ) {
```
Comment (janvanmansum) [inline src/main/java/edu/harvard/iq/dataverse/api/imports/ImportDDIServiceBean.java]: Remove commented out code if it is no longer useful.

File: src/test/java/edu/harvard/iq/dataverse/export/OpenAireExportUtilTest.java [hunk 2]
```diff
@@ -906,8 +904,8 @@ public void testWriteAccessRightElement() throws XMLStreamException, FileNotFoun
         xmlw.close();
         Assert.assertEquals("<rightsList>"
                 + "<rights rightsURI=\"info:eu-repo/semantics/closedAccess\" />"
-                + "<rights rightsURI=\"https://creativecommons.org/publicdomain/zero/1.0/\">"
-                + "CC0 Waiver</rights></rightsList>",
+                + "<rights rightsURI=\"https://creativecommons.org/publicdomain/zero/1.0/\" />"
```
Comment (janvanmansum) [inline src/test/java/edu/harvard/iq/dataverse/export/OpenAireExportUtilTest.java]: In the old version there was a human-readable name in the element text. Is that no longer useful?

**Review by janvanmansum — CHANGES_REQUESTED** [review]
LGTM. A couple of minor points, see comments below.

File: src/test/java/edu/harvard/iq/dataverse/export/OpenAireExportUtilTest.java (hunk 2 above)
Comment (mderuijter) [reply src/test/java/edu/harvard/iq/dataverse/export/OpenAireExportUtilTest.java]: it would cause the unit test to fail

Comment (janvanmansum) [reply src/test/java/edu/harvard/iq/dataverse/export/OpenAireExportUtilTest.java]: And in the old version it didn't fail, why?

**Pull request merged**